client-grpc/src/grpc.vehicle.rs

```diff
@@ -2,6 +2,7 @@
 #[derive(Eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct Vehicle {
+    /// UUID v4
     #[prost(string, tag="1")]
     pub id: ::prost::alloc::string::String,
     #[prost(message, optional, tag="2")]
```

Comment:
Vehicle should also have schedule field - as it can have it's own maintenance windows etc.